styles.css

```diff
@@ -0,0 +1,523 @@
+    justify-content: center;
+    align-items: center;
+}
+.section-intro div h2{
+    text-align: center;
+    color: var(--paars);
+    padding-bottom: 30px;
+}
+.section-intro div p{
+    padding-top: 70px;
+}
+/*info-blocks section */
+.section-info-blocks{
+    position: relative;
+    background-color: var(--lichtgrijs);
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 180px 20px 60px 20px;
+    z-index: 0;
+
+}
+.info-block{
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 60px;
+
+    align-items: center;
+    justify-content: space-between;
+    z-index: 2;
+
+}
+.info-block:nth-child(even){
+    flex-direction: row-reverse;
+}
+
+.section-info-blocks div article div{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    max-width: 300px;
+    height: 240px;
+    margin: 0 20px;
+}
+.section-info-blocks div article div a{
+    font-weight: bold;
+    letter-spacing: 4px;
+    position: relative;
+}
+.section-info-blocks div article div a:hover{
+    color: var(--paars);
+    cursor: pointer;
+}
+.section-info-blocks div article div a::after{
+    content: "";
+    display: block;
+    margin-left: 169px;
+    margin-top: -11px;
+    width: 10%;
+    height: 2px;
+    background-color: var(--paars);
+    transition: width ease-in-out 0.3s;
+}
+.section-info-blocks div article div a:hover::after{
+    width: 40%;
+
+}
+.image-wrapper {
+    flex-basis: 550px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+.section-info-blocks div article span img{
+    width: 100%;
+    box-shadow: 0px 10px 15px rgba(110, 110, 110, 0.3);
+}
+/*info-squares section */
+.section-info-squares{
+    padding: 220px 20px 80px 20px;
+}
+.section-info-squares div{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    gap: 30px 25px;
+}
+
+.info-square{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: flex-start;
+    width: 320px;
+    height: 300px;
+    border: 2px solid var(--paars);
+    padding: 15px 30px 30px 30px;
+    box-shadow: 0 10px 15px rgba(110, 110, 110, 0.2);
+    background-color: white;
+}
+.info-square img{
+    width: 60px;
+}
+.info-square h3{
+    color: var(--paars);
+    font-size: 1.3rem;
+}
+.info-square p{
+    font-size: 0.85rem;
+    color: #939393;
+}
+
+/*Contact form */
+
+#section-contact-form{
+    width: 101%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 50px 20px;
+}
+#section-contact-form div{
+    padding-top: 40px;
+
+}
+#section-contact-form div h1, #section-contact-form div p{
+    color: white;
+    text-align: center;
+    padding-bottom: 25px;
+}
+form{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    width: 700px;
```

Comment:
Hmmmmmm niet zo'n goed idee. De <div> die er omheen staat bepaald de maximale breedte al (als dat te breed is moet je die even aanpassen), dus je `form` wil dan binnen die kaders altijd `width: 100%` zijn.